nbd/src/main.cpp

```diff
@@ -95,6 +97,112 @@ static void Usage() {
         << std::endl;
 }
 
+// use for record image and device info to auto map when boot
+static void AddRecord() {
+    std::fstream fs;
```

Comment:
You can use `std::ifstream` to replace `std::fstream` and open with `std::fstream::in`.

Reply:
> You can use `std::ifstream` to replace `std::fstream` and open with `std::fstream::in`.

because fs reused latter in this function as std::fstream::app